### DISCUSSION TITLE
Hidden files in filebrowser mode

### DISCUSSION THREAD

**Discussion by ghost** [davatorium/rofi#1480] [Ideas/Feature Requests]
From a quick look at `filebrowser.c`, lines `233-235`, it seems that file names beginning with a dot are unconditionally ignored. I am thinking of just removing these lines for myself but I would like to know:
1) Whether there is any known gotcha behind just removing these three lines and
2) Whether a patch for this would be welcome (with the option to toggle the behaviour, obviously).

**Comment by boxallar** [comment]
Has this been changed?
If so, what is the toggle shortcut? I have tried '.' and Ctrl+H etc., but have not found the solution.

**Reply by DaveDavenport** [reply]
I don´t think anything changed on this.  Sorry for forgetting to reply, I must have missed this one. 
A patch that adds a key to toggle this would be much appreciated.

**Comment by ghost** [comment]
I added the option in #1716. There is no keybinding to change it on the fly at the moment though, it is a configuration option.
